tasks/main.yml

```diff
@@ -1,21 +1,19 @@
 ---
 - include: node_dependencies.yml
 
-# This assumes that the packages are available in an existing Apt repo
-- name: Install Vertica packages
-  apt: name="{{item}}" state=present
-  with_items: "{{vertica_packages}}"
+# This assumes that the packages are not available and the packages are downloaded already
+- name: Install Vertica
+  apt:  deb="{{vertica_package_location}}" state=present
+  sudo: true
 
 - include: node_users.yml
 - include: node_setup.yml
 
 - include: cluster.yml
 
-
-# A DB must exist before the node can startup.
 - include: create_db.yml
 
-- meta: flush_handlers
+# - meta: flush_handlers
```

Comment:
Doesn't look like the handlers are used at all. That means you can remove this line as well as the handlers.